assets/app/views/settings.html

```diff
@@ -8,16 +8,16 @@
       <h2>General information</h2>
       <!-- TODO make these fields editable - probably with a cog icon that flips into inline-editing -->
       <dl class="dl-horizontal left">
-        <dt>Name:</dt>
-        <dd>{{projectName}}</dd>
-        <dt>Display name:</dt>
-        <dd ng-if="project | displayName: true">{{project | displayName: true}}</dd>
-        <dd ng-if="!(project | displayName: true)"><em>No display name</em></dd>
-        <dt>Description:</dt>
-        <dd ng-if="project | description">
+        <dt class="name">Name:</dt>
```

Comment:
Fixed by the use of the index of every given `dd` inside the `dl`. I tried to find something like the good old `dt:contains("Name") ~ dd` but the webdriver doesn't seem to provide anything like that. Could be solved by the use of an xpath selector, but it seems to be an overkill.